src/Http/Controllers/PostController.php

```diff
@@ -2,20 +2,23 @@
 
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helpers. It's overkill to me.